crates/rpc/rpc/src/eth/signer.rs

```diff
@@ -36,15 +36,26 @@ pub(crate) trait EthSigner: Send + Sync {
 
     /// Encodes and signs the typed data according EIP-712. Payload must implement Eip712 trait.
     fn sign_typed_data(&self, address: Address, payload: &TypedData) -> Result<Signature>;
+
+    /// Clone boxed instance of EthSigner
+    fn clone_box(&self) -> Box<dyn EthSigner + Send + Sync>;
 }
 
 /// Holds developer keys
+#[derive(Clone)]
 pub(crate) struct DevSigner {
     addresses: Vec<Address>,
     accounts: HashMap<Address, SecretKey>,
 }
 
 impl DevSigner {
+    pub fn new() -> DevSigner {
```

Comment:
```suggestion
    pub fn random() -> DevSigner {
       Self::random_signers(1)
    }   
```

and `random_signers(count: usize)`